Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -428,6 +352,108 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
 
         #endregion
 
+        #region Private Helpers
+
+        private IProjectEntry RemoveEntry(Uri documentUri) {
+            var entry = _projectFiles.RemoveEntry(documentUri);
+            _openFiles.Remove(documentUri);
+            return entry;
+        }
+
+        private async Task DoInitializeAsync(InitializeParams @params) {
```

Comment:
We could move it to `Server.Initialize.cs` and `Server.Extensions.cs`

Reply:
If you think it's worth splitting it up at that fine-grained a level, sure. I'm personally not a fan, but happy to follow your direction here

Reply:
OK. Just making files reasonably small to scroll through :-)